cloudify_azure/utils.py

```diff
@@ -441,9 +441,23 @@ def handle_properties_keys(data):
 
 
 def get_client_config(properties):
+    # ctx.plugin
+    plugin_props = getattr(ctx.plugin, 'properties', {})
+
+    # ctx.node.properties
     client_config = properties.get('client_config', {})
     azure_config = properties.get('azure_config', {})
 
+    final_client_config = {}
+    if plugin_props:
+        for k, v in plugin_props.items():
+            if isinstance(v, dict):
+                final_client_config[k] = v.get('value')
+            else:
+                final_client_config[k] = v
+        client_config.update(final_client_config)
+        azure_config.update(final_client_config)
```

Comment:
here we don't want to override the node level properties from the plugin properties level , it should be the other way around [ we can override the plugin properties from the node level ]